health/omsagent-template.yaml

```diff
@@ -93,7 +93,7 @@ data:
      #Kubernetes health
      <source>
       type kubehealth
-      tag oms.api.KubeHealth.ReplicaSet
+      tag kubehealth.ReplicaSet
```

Comment:
do we need these template files, now that we are releasing health with default on ?

Reply:
I will fix them as part of the ignite agent PR